app/components/agent/Index/quick-start.js

```diff
@@ -7,8 +7,6 @@ import classNames from 'classnames';
 import Emojify from '../../shared/Emojify';
 import Panel from '../../shared/Panel';
 
-import 'highlight.js/styles/atom-one-light.css';
```

Comment:
Webpack couldn't compile it, and it's the only file that requires CSS outside of `main.css` so figured I'd move it in there. But I'm not sure where we're heading with stuff like this? 

I was going to do a follow-up PR w/ `ExtractTextPlugin` and a proper external CSS file anyways, rather than the JS-loaded version we use now. And so this might possibly work again in that setup.